README.md

```diff
@@ -1,23 +1,146 @@
+    <br />
+    <br />
+    <a href="https://laperette-client.herokuapp.com/">View Demo</a>
+    ·
+    <a href="https://github.com/laperette/laperette/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/laperette/laperette/issues">Request Feature</a>
+  </p>
+</p>
+
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+- [About the Project](#about-the-project)
+  - [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+  <!-- - [Usage](#usage) -->
+- [Roadmap](#roadmap)
+- [Contributing](#contributing)
+- [License](#license)
+- [Contact](#contact)
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+<!-- [![Product Name Screen Shot][product-screenshot]](https://example.com) -->
+
+### Built With
+
+- [Typescript](https://www.typescriptlang.org/)
+- [React](https://reactjs.org//)
+- [Koa](https://koajs.com/)
+- [PostgreSQL](https://www.postgresql.org/)
+- [Knex](http://knexjs.org/)
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- Install Docker
+- Install yarn
+
+### Installation
+
+1. Clone the repo
+
+```sh
+git clone https://github.com/laperette/laperette
+```
+
+2. Install the packages
 
 ```sh
 yarn install
 ```
 
-## Start local development services:
+3. Launch the local database accessible at `postgres://postgres:postgres@localhost:5430/laperette`
 
 ```sh
-docker-compose up
+docker-compose up -d
 ```
 
-This command will start 3 services:
+4. Run the migrations
 
-- app_db: a postgre sql database, accesssible on the local host at `postgres://postgres:postgres@localhost:5430/laperette`
-- app_server: the node server, started with nodemon
-- app_client: the react app, started with react-scripts
+```sh
+yarn server migrate
+```
 
-## Run migrations
+5. Launch the server
 
 ```sh
-docker-compose run app_server yarn server migrate
+cd packages/server
+yarn start-dev
 ```
+
+6. Launch the client
+
+```sh
+cd packages/client
+yarn start-dev
+```
+
+<!-- USAGE EXAMPLES -->
+
+## Usage
+
+<!-- Use this space to show useful examples of how a project can be used. Additional screenshots, code examples and demos work well in this space. You may also link to more resources.
+
+_For more examples, please refer to the [Documentation](https://example.com)_ -->
+
+<!-- ROADMAP -->
+
+## Roadmap
+
+See the [open issues](https://github.com/laperette/laperette/issues) for a list of proposed features (and known issues).
+
+<!-- CONTRIBUTING -->
+
+## Contributing
+
+Any contributions you make are **greatly appreciated**, especially styling!
+
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
```

Comment:
I think you can remove this..

Reply:
I thought I would leave it as a guidance for potential other developers that would join the project :))